### PR TITLE
update docs on Polkadot/Cosmos comparison

### PR DESCRIPTION
**Pull request by adiraviraj**
this PR updates the docs to reflect more up to date info on the Polkadot/Cosmos comparison section. most of the edits pertain to correcting explanations on the topology of the Cosmos ecosystem and other edits to terminology.

### CODE REVIEW

**Comment by DrW3RK** [comment]
Let us know when this is ready for review. Thanks for taking time to update this info!

**Comment by adiraviraj** [comment]
thank you. its ready for review @DrW3RK

File: docs/learn/learn-comparisons-cosmos.md [hunk 1]
```diff
@@ -55,18 +52,9 @@ Polkadot has [bridge parachains](learn-bridges.md) that offer two-way compatibil
...
-(non-transfer messages are relayed but their state not stored in the Hub).
+Cosmos is a network of blockchains built using [CometBFT](https://cometbft.com/) as the consensus engine, [Cosmos SDK](https://docs.cosmos.network/) as the VM, and [IBC](https://ibcprotocol.dev/) which allows chains to interoperate with one another.
 
-Zones monitor the state of the Hub with a light client, but the Hub does not track zone states.
-Zones must use a deterministic finality algorithm (currently, all use Tendermint) and implement the
-IBC interface to be able to send messages to other chains through the Hub.
-
-Cosmos can also interact with external chains by using "peg zones", which are similar to bridged
-parachains.
+IBC leverages light clients that can keep track of the consensus of a counterparty chain. For example, when chains A and B want to talk to one another, chain A uses its light client of B to verify messages sent from chain B, and vice versa. IBC is [currently live](https://app.trustless.zone/?from=POLKADOT&to=OSMOSIS) on Polkadot and Kusama. Work is ongoing to implement IBC to Ethereum and it's layer 2s.
```
Comment (profradha) [inline docs/learn/learn-comparisons-cosmos.md:57]: Thanks for adding the Polkadot <-> Osmosis trustless bridge info

File: docs/learn/learn-comparisons-cosmos.md [hunk 2]
```diff
@@ -87,7 +75,7 @@ Second, having the capacity to extend the chain with unfinalized blocks allows o
 perform extensive availability and validity checks to ensure that no invalid state transitions make
 their way into the final chain.
 
-Cosmos (both the Hub and the zones) uses Tendermint consensus, a round-robin protocol that provides
+Cosmos chains uses Tendermint consensus, a round-robin protocol that provides
```
Comment (profradha) [inline docs/learn/learn-comparisons-cosmos.md]:
```suggestion
Cosmos chains use Tendermint consensus, a round-robin protocol that provides
```

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**